spec/game_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../classes/game'
+
+describe 'Testing the games class implementation' do
+  before(:each) do
+    @games = Game.new(true, '01-01-2090')
+  end
+
+  it 'Returns the class instance' do
+    expect(@games).to be_instance_of Game
+  end
+
+  it 'Return the superclass of Game' do
```

Comment:
Team member #3

Based on the requirement you need to write tests for your all methods. I can see that you didn't write a test for `can_be_archived?()`. Kindly add test for this method as well :)

> So, in this test:
> should return true if the parent's method returns true AND if last_played_at is older than 2 years
> otherwise, it should return false